src/containers/Network/NetworkTab.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
```

Comment:
The refactors to TypeScript/Hooks should be a separate PR from the new feature, to make reviewing easier

Reply:
This was done at the same time because there was a lot of code that was shared with the validator list page.

Reply:
This is also a new component that was created to prevent the same code from being copied in 3 places.  Was two before the new tab.

Reply:
The refactors can still be in a separate PR, though.

Reply:
Fair.